### PR TITLE
Load backgrounds asynchronously

### PR DESCRIPTION
**Pull request by biyeun**
Loads backgrounds on all pages asynchronously instead of synchronous with the rest of the page's images.

@orangejenny cc @calellowitz

### CODE REVIEW

**Comment by biyeun** [comment]
@orangejenny fyi ✅

**Comment by czue** [comment]
:horse_racing:

File: templates/prelogin/base.html [hunk 1]
```diff
@@ -243,6 +243,22 @@ <h3 class="modal-title">
...
         {% endcompress %}
 
+        {# Blazy for async loading images #}
+        {% compress js %}
+            <script type="text/javascript" src="{% static 'blazy/blazy.js' %}"></script>
+            <script type="text/javascript">
+              var hqLazy = new Blazy({
+                container: 'body',
+                success: function(element) {
+                    setTimeout(function(){
```
Comment (orangejenny) [inline templates/prelogin/base.html]: nit, but the spacing here is funky, hard to tell what's in the function

**Comment by gcapalbo** [comment]
@biyeun when deploying today with this pr the compress step failed:

    Executing task '_do_compress'
    sudo: ... python manage.py compress --force -v 0
    out: CommandError: An error occured during rendering ... corehq/apps/prelogin/templates/prelogin/home.html: sequence item 1: expected string or Unicode, NoneType found

for now i kept the hq prelogin submodule pointing to yesterday's ref

**Pull request merged**